.github/workflows/inspektor-gadget.yml

```diff
@@ -466,11 +466,6 @@ jobs:
     # docs/ci.md for further details.
     if: needs.check-secrets.outputs.aro == 'true'
     runs-on: ubuntu-latest
-    permissions:
-      # allow publishing container image
-      # in case of public fork repo/packages permissions will always be read
-      contents: read
-      packages: write
```

Comment:
I don't see why it shouldn't work, but could you please try a merge and a release (I usually comment the release step to do it) on a fork to be sure it is working properly?

Reply:
A green CI run with tag can be found [here](https://github.com/mqasimsarfraz/inspektor-gadget/actions/runs/3090761608).